src/node/utils/Cli.js

```diff
@@ -40,12 +40,12 @@ for ( var i = 0; i < argv.length; i++ ) {
   }
 
   // Override location of settings.json file
-  if ( prevArg == '--sessionkey' || prevArg == '-k' ) {
```

Comment:
Removing an argument ruins backward compatibility.
Was this argument so broken that removing it is not going to break any installation?

I am 100% ok merging the doc changes, but would like some details on this change.